doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
That's probably a fair usage model.  When a validator's master key changes then the `validators.txt` must be adjusted to include the new public master key.  When that edit happens then the revoked public masker key can be removed from the `validators.txt`.